Poisonous-Potato/JTK-Website
# Call Footer API

### Description
Create Rest API Caller and call footer API

### How to Test
1. Run the portal
2. Run the CMS
3. Navigate to http://localhost:3000
4. Chack the footer. It should be the data from CMS
